### ISSUE TITLE
Lock focus chain within settings pane

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#238]
I'm able to get outside of settings pane focusable element using `tab` key.

After doing so I have no way to enter into the settings pane once again. General principle is that focus should be locked on focusable elements within focus pane only.

**Comment by CKFinderBOT** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
